### ISSUE TITLE
"Unable to read index" error

### ISSUE THREAD

**Issue by mdtanrikulu** [ibcn-cloudlet/dianne#8]
Hi,

The problem is when I run `./gradlew build` I'm getting;
```
Error  : dianne :Unable to read index at URL 'https://raw.githubusercontent.com/osgi/osgi.enroute/master/cnf/distro/index.xml': java.io.EOFException
```
I checked given url and took me [osgi.enroute](https://github.com/osgi/osgi.enroute) repository from `osgi`.

Then I see that you already update `master` tag as `v1.0.0` in the commit 497a700de4ba02ec453e4ac9d296fced91a818f6 
Can you add some information into your documentation about v5.0.0 and this expired link?

**Comment by tverbele** [comment]
Thanks for spotting this! Instead of adding documentation, I have fixed this on top of the old v0.5.0 tag and updated the tag. If you checkout v0.5.0 again it should work now.

**Comment by mdtanrikulu** [comment]
Thanks it works.
